### DISCUSSION TITLE
HA si o no? consiglio

### DISCUSSION THREAD

**Discussion by BaXaras** [indomus/forum#1851] [GENERALE - Discussioni generiche]
Buongiorno a tutti, vorrei un vostro consiglio per un dilemma che mi attanaglia da anni, utilizzo OpenHab da anni per la mia casa mi trovo bene ma ho sempre rosicato a vedere la semplicità di HA per molti aspetti, l'unica cosa che mi ferma è la possibilità di gestirlo da remoto per il cloud a pagamento o aprire una porta del router di casa, cosa che non mi piace tanto per ovvi motivi di sicurezza. Ho bisogno di un vostro consiglio o una vostra esperienza sulla sicurezza di questo strumento o se c'è un altra possibilità che non conosco per raggiungere il mio Hub da remoto.
Grazie ancora e complimenti per le guide fantastiche del sito!

**Comment by snakuzzo** [comment]
Ciao,
Se sai quello che fai, aprire porte sul router non è affatto un "ovvio problema di sicurezza". Se fosse così non avrebbe nemmeno senso mettere a disposizione questa funzionalità sui dispositivi di rete.
Nel caso di HA dovresti aprire una porta a piacere e fine. HA prevede accesso anche con 2FA, quindi direi che, salvo buchi di sicurezza dell'applicazione, puoi stare sereno.

**Comment by KC-inDomus** [comment]
come ti spiega anche il collega Diego, quello della sicurezza è un non problema. passa ad HA, smetti di rosicare. 👍
